app/views/rubric_criteria/index.html.erb

```diff
@@ -18,15 +18,23 @@
   <div class='heading_buttons'>
     <%= link_to t('criteria.add_criterion'),
       'javascript:void(0);',
-      onclick: "if(document.getElementById('new_criterion') != null) {
+      onclick: "var criteriaSelectionText = jQuery('#marking-scheme-criteria-dropdown :selected').val();
+      if (criteriaSelectionText !== 'RubricCriterion' && criteriaSelectionText !== 'FlexibleCriterion') {
+        alert('Please select a criterion from the dropdown menu.');
```

Comment:
Eh, an alert is fine for now. This message should also be internationalized, though.